### PR TITLE
home layout

### PR DESCRIPTION
**Pull request by HaNguyen107**

### CODE REVIEW

File: resources/views/layouts/master.blade.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    @if (Session::has('success'))
+    <div class="alert alert-success"> 
+        {{Session::get('success')}}
+    </div>
+    @endif
+    @yield('content')
+    @yield('footer')
+</div>
+</body>
+</html>
```
Comment (huytbt) [inline resources/views/layouts/master.blade.php]: Need enter after line

File: resources/views/home/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+@stop
+@section('content')
+
+@stop
+
+@section('footer')
+<footer>
+
+</footer>
+@stop
```
Comment (huytbt) [inline resources/views/home/index.blade.php]: Need enter after line

File: resources/views/layouts/master.blade.php (hunk 1 above)
Comment (HaNguyen107) [reply resources/views/layouts/master.blade.php]: ok a

**Review by huytbt — APPROVED** [review]
ok

**Pull request merged**